robotic_arm_control/python/imuAnalyzer.py

```diff
@@ -101,10 +115,13 @@ def elbowImuCallback(self, data):
 		Args:
 				data (Vectornav): The IMU data
 		"""
-		# self.linkAngleBuffers[2]
-		# self.linkVelocityBuffer[2]
-		pass
 
+		relAngle = float(self.quat2eul(data.imu.orientation) - self.zeroAngles[2])
+		self.linkAngleBuffers[2].update(relAngle)
+
+		# assume all link starting velocities are 0
+		self.linkVelocityBuffers[2].update(data.imu.angular_velocity.z)
```

Comment:
This is good, one thing I forgot to mention was that I want to store velocities as RPM so we can set the speed on the arduino by passing in desired RPM which should be within a specific range that we can clamp between. I can do that but this is solid.